README.md

```diff
@@ -1,15 +1,31 @@
 # React Accordion
-A really lightweight, symantic, super simple accordion for React
-
-## Roadmap
-
-- [x] Basic iteration over items to product accordion frame
-- [x] Multiple/Single expanded option
-- [ ] Default theme
-- [x] Write test for passing open prop
-- [x] Support childless item
-- [ ] CI/CD
-- [x] GitHub Pages Demo
-- [ ] Contributing/Security documents
-- [ ] Deploy to NPM
-- [ ] Cleanup old preview versions once they are superseded or merged
+A really lightweight, accessible, semantic, super simple accordion for React
+
+## TODO
+Put some badges including maybe NPM bundle size
+
+## Installation
+TODO
```

Comment:
Need to wait until we have created an NPM package and chosen a name until we can fill this bit out.
Let's just keep this PR open and add to it while the other tick-boxes are finished.